src/components/task-item/TaskItem.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./taskitem.css"
+
+export default function TaskItem(task) {
+    const {  title } = task;
+
+    return (
+        <div className="task-item" onClick={()=>{console.log("clicked")}}>
```

Comment:
you should not console log click and If it is button you can show it in button tag.

Reply:
actually wanted to make the TaskItem as whole clickable and it contains icons as well so I resorted to this approach.